src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -209,94 +377,216 @@ static void OnFlyoutBehaviorChanged(BindableObject bindable, object oldValue, ob
+		/// <returns>The foreground color for the tab bar.</returns>
 		public static Color GetTabBarTitleColor(BindableObject obj) => (Color)obj.GetValue(TabBarTitleColorProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetTabBarTitleColor']/Docs/*" />
+
+		/// <summary>
+		/// Sets the title color for the tab bar. 
+		/// If the property is unset, the TitleColor property value will be used.
+		/// </summary>
+		/// <param name="obj">The object to which the color is set.</param>
+		/// <param name="value">The title color for the tab bar.</param>
 		public static void SetTabBarTitleColor(BindableObject obj, Color value) => obj.SetValue(TabBarTitleColorProperty, value);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetTabBarUnselectedColor']/Docs/*" />
+		/// <summary>
+		/// Gets the title color for the tab bar. 
+		/// </summary>
+		/// <param name="obj">The object to which the color is set.</param>
+		/// <returns>The title color for the tab bar.</returns>
 		public static Color GetTabBarUnselectedColor(BindableObject obj) => (Color)obj.GetValue(TabBarUnselectedColorProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetTabBarUnselectedColor']/Docs/*" />
+
+		/// <summary>
+		/// Sets the unselected color for the tab bar. 
+		/// If the property is unset, the UnselectedColor property value is used.
+		/// </summary>
+		/// <param name="obj">The object to which the color is set.</param>
+		/// <param name="value">The unselected color for the tab bar.</param>
 		public static void SetTabBarUnselectedColor(BindableObject obj, Color value) => obj.SetValue(TabBarUnselectedColorProperty, value);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetTitleColor']/Docs/*" />
+		/// <summary>
+		/// Gets the unselected color for the tab bar. 
+		/// </summary>
+		/// <param name="obj">The object to which the color is set.</param>
+		/// <returns>The unselected color for the tab bar.</returns>
 		public static Color GetTitleColor(BindableObject obj) => (Color)obj.GetValue(TitleColorProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetTitleColor']/Docs/*" />
+
+		/// <summary>
+		/// Sets the color used for the title of the current page.
+		/// </summary>
+		/// <param name="obj">The object to which the color is set.</param>
+		/// <param name="value">The color used for the title of the current page.</param>
 		public static void SetTitleColor(BindableObject obj, Color value) => obj.SetValue(TitleColorProperty, value);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetUnselectedColor']/Docs/*" />
+		/// <summary>
+		/// Gets the color used for the title of the current page.
```

Comment:
This might've been mixed up with the GetTitleColor

Reply:
Fixed.